kubernetes_yaml/chatbot/README.md

```diff
@@ -0,0 +1,38 @@
+## Simple chatbot
+
+This folder holds the resource definitions to launch a chatbot.
+Environment variables `MODEL_ENDPOINT` must be provided.
+Optionally, `MODEL_ENDPOINT_BEARER`, and `MODEL_NAME` can be provided.
+
+Update the deployment as necessary and
+run this from the root of the repository
+
```

Comment:
maybe just add a line in here describing the line that must be changed and an exampl